lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -222,6 +224,15 @@ def enrolled_students_features(course_key, features):
     if include_team_column:
         students = students.prefetch_related('teams')
 
+    def extract_attr(student, feature):
```

Comment:
@regisb Do you know why this was not previously needed for "location" and "mailing_address", which aren't actually filled out (why didn't they throw an error also)?